gluten-ut/spark35/src/test/scala/org/apache/spark/sql/GlutenBloomFilterAggregateQuerySuite.scala

```diff
@@ -106,11 +107,52 @@ class GlutenBloomFilterAggregateQuerySuite
           assert(
             collectWithSubqueries(df.queryExecution.executedPlan) {
               case h if h.isInstanceOf[HashAggregateExecBaseTransformer] => h
-            }.size == 0,
+            }.size == 2,
```

Comment:
Could you document this change?

Reply:
No need to add documents. Will change the test name to `Test bloom_filter_agg offloaded with filter fallen back` to clarify in next patch